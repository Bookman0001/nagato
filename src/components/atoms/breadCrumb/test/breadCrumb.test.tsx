import { shallow } from 'enzyme'

import BreadCrumb from '../'

test('BreadCrumb', () => {
  const component = shallow(
    <BreadCrumb
      contents={[
        { label: 'TOP', link: '/' },
        { label: 'SECOND', link: '/' },
        { label: 'THIRD', link: '/' },
      ]}
    />
  )
  expect(component).toMatchSnapshot()
})
