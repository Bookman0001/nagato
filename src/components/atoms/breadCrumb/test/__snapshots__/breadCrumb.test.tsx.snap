// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`BreadCrumb 1`] = `
<styled.div>
  <Link
    href="/"
    key="0"
  >
    <styled.div>
      TOP
      <span>
        &gt;
      </span>
    </styled.div>
  </Link>
  <Link
    href="/"
    key="1"
  >
    <styled.div>
      SECOND
      <span>
        &gt;
      </span>
    </styled.div>
  </Link>
  <Link
    href="/"
    key="2"
  >
    <styled.div>
      THIRD
      <span>
        &gt;
      </span>
    </styled.div>
  </Link>
</styled.div>
`;
